__tests__/app.test.js

```diff
@@ -33,4 +34,20 @@ describe("/api/topics", () => {
                 expect(body.msg).toBe("path not found");
             });
     });
+});
+
+describe("/api", () => {
+    test("GET:200 responds with an object describing all the available endpoints", () => {
+        return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({ body }) => {
+                const readFile = fs.readFile(`${__dirname}/../endpoints.json`);
+                return Promise.all([body.api, readFile]);
+            })
+            .then(([api, fileContents]) => {
+                const apiData = JSON.parse(fileContents);
+                expect(api).toEqual(apiData);
+            });
```

Comment:
Nicely done! We could simplify it further by just requiring in the .json file and expecting the response to equal it, but I'll leave it up to you if you wanna refactor 👍

Reply:
oh cool! didn't know you could just do that, thank you!